omp/__init__.py

```diff
@@ -7,10 +7,11 @@
 import sys
 
 try:
+    # there may be an environ modification when loading config
     from pythran import config
-    cxx = config.cfg.get('user', 'cxx')
 except ImportError:
-    cxx = 'c++'
+    pass
+cxx = os.environ.get('CXX', 'c++')
```

Comment:
I agree with your arguments but disagree on the conclusion ;-)

The problem is "how can the user state which compiler I can use to find an openmp runtime".
CXX is not always defined, so it's not an option
c++ is not always a valid symlink, so it's not an option

The best way I had was to rely on pythranrc, or CXX, or c++ in that
order.